### PR TITLE
info: add cgroups2

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/containers/libpod/issues/4080

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: libpod/info.go [hunk 1]
```diff
@@ -29,6 +30,11 @@ func (r *Runtime) hostInfo() (map[string]interface{}, error) {
 	info["arch"] = runtime.GOARCH
 	info["cpus"] = runtime.NumCPU()
 	info["rootless"] = rootless.IsRootless()
+	unified, err := cgroups.IsCgroup2UnifiedMode()
+	if err != nil {
+		return nil, errors.Wrapf(err, "error reading cgroups mode")
+	}
+	info["cgroupv2"] = unified
```
Comment (AkihiroSuda) [inline libpod/info.go]: cgroup2? cgroupv2?

Comment (mheon) [reply libpod/info.go]: Maybe we should set this to a string instead - "cgroupmode" = "v2" or "v1"?

Comment (AkihiroSuda) [reply libpod/info.go]: legacy/hybrid/unified?

Comment (baude) [reply libpod/info.go]: i like the idea from @mheon

Comment (giuseppe) [reply libpod/info.go]: the issue is that currently we have only code to detect the unified mode (that is cgroups v2).  Should we probably rename it to "cgroups-unified-mode"?

Comment (rhatdan) [reply libpod/info.go]: Users understand cgroups V1 and V2,  I don't want the "unified" mode exposed.  When I see unified, I get confused and think it is a mode that covers both V1 and V2.  

"CgroupVersion"  = V2

That way 5 years from now when V3 is all the rage ...

Comment (giuseppe) [reply libpod/info.go]: fixed to be `"CgroupVersion" = "v1" | "v2"`

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**